server/server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const app = express();
+const port = 4000;
+const mongoose = require('mongoose');
+const db = require('../database/database.js');
+
+app.use('/:id', express.static(__dirname + '/../client/dist'));
+
+// open mongoose connection
```

Comment:
Is there any specific reason for having this code here instead of in your database folder? This seems like code that might make more sense being placed in your database folder.

Reply:
There's no good reason other than I was having issues with my other service's test! I will move it back to the db file and resolve any issues!